contracts/contracts/L2/submitter/Submitter.sol

```diff
@@ -217,27 +170,32 @@ contract Submitter is Initializable, Semver, ISubmitter, RollupMessage {
         view
         returns (address, uint256, uint256)
     {
-        // update epoch
-        address[] memory sequencers = IL2Sequencer(L2_SEQUENCER_CONTRACT)
-            .getSequencerAddresses(false);
+        require(sequencerHistory.length > 0, "invalid sequencer");
+        uint256 start = sequencerHistory[sequencerHistory.length - 1].timestamp;
+
+        if (
+            epochHistory.length > 0 &&
```

Comment:
no need to check `epochHistory.length > 0`

Reply:
If not check whether length is 0, it will throw an error when epochHistory.length = 0. But I want it do nothing when it is 0